jenkins/test/testsuite/Jenkinsfile

```diff
@@ -62,26 +59,60 @@ pipeline {
             steps {
                 script {
                     basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"
+                    sh "wget ${basePath}/builds/manifest.yml"
```

Comment:
Is this for debugging?

Reply:
It's for fetching the build manifest.

Reply:
I mean why is it needed?

```
    test_manifest = TestManifest.from_path(test_manifest_path)
    bundle_manifest = BundleManifest.from_urlpath("/".join([args.path.rstrip("/"), "dist/manifest.yml"]))
    build_manifest = BuildManifest.from_urlpath("/".join([args.path.rstrip("/"), "builds/manifest.yml"]))
```

This fetches everything from the `basePath` URL.

Reply:
Answered my own question, we have code in publishTestResults that loads this manifest. Similarly, it should be loading it from the remote URL. Given that `readYaml` doesn't support it I think the "right thing to do" (TM) would be that `./test.sh ...` produces a `results.yml` that gets loaded at the end of the tests and then the publication location is derived from those test results, not from the build YAML. Please open an issue for this.

Reply:
Created https://github.com/opensearch-project/opensearch-build/issues/1113